src/main/gen/org/amahi/anywhere/R.java

```diff
@@ -0,0 +1,7 @@
+/*___Generated_by_IDEA___*/
+
+package org.amahi.anywhere;
+
+/* This stub is only used by the IDE. It is NOT the R class actually packed into the APK */
+public final class R {
+}
```

Comment:
What's this file? Is it necessary?

Reply:
@chirag-jn I don't know about this file. when i created 1st pull request i get a error in github then a went to git bush and commit all files and then i created pull request and github did not show me any eror. I just edited string.xml files nothing more than that.

Reply:
Please check the files again. We don't need these files.